tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/LGOptionTest.lg

```diff
@@ -0,0 +1,13 @@
+﻿> !# @strict = true
+> !# @replaceNull = The ${path} is undefined  
+> !# @lineBreakStyle = markdown
```

Comment:
I would love to see some tests of how this works with imports of another file.  Is it options and I override subsequent imports?  Is it options in an import are used there, but not in the parent?  I personally like the semantic where is A imports B that within A only its options are used and within B any options specified override those in A for the templates in B.

Reply:
more tests about merge options are added.